waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/client/ThriftMetastoreClientManagerIntegrationTest.java

```diff
@@ -36,7 +36,7 @@
 
   private final HiveCompatibleThriftHiveMetastoreIfaceFactory hiveCompatibleThriftHiveMetastoreIfaceFactory = new HiveCompatibleThriftHiveMetastoreIfaceFactory();
   private final HiveConf hiveConf = new HiveConf();
-  private final int connectionTimeout = 10;
+  private final int connectionTimeout = 600;
```

Comment:
For better understanding, why did we need to increase this timeout ?

Reply:
Lesser timeout causes ConnectException (timeout, not "refused" ) on windows machine when running tests.

Reply:
I wonder if its a Windows issue where some code path takes longer or if its just a slow machine? @chergey what's the lowest value you can set this to that lets the test pass for you?